.travis.yml

```diff
@@ -8,35 +8,54 @@ cache:
-  - INCLUDE="test_module,second_module" UNIT_TEST="1"
-  - VERSION="7.0" INCLUDE="test_module,second_module" ODOO_REPO="OCA/OCB"  # ODOO_REPO usage example
-  - TESTS="1" INCLUDE="test_module,second_module" ODOO_REPO="OCA/OCB" MQT_TEMPLATE_DB='mqt_odoo_template_core' MQT_TEST_DB='mqt_odoo_core'
-  - VERSION="6.1" INCLUDE="test_module,second_module"
-  - LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="21" TRAVIS_PULL_REQUEST="false" # Use main pylint config file
-  - VERSION=master EXCLUDE="broken_no_access_rule" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="34" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
-  - VERSION="7.0" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="32" TRAVIS_PULL_REQUEST="true"  # To check pylint_conf of PR's with old api
-  - VERSION="" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="21" TRAVIS_PULL_REQUEST="false"  # To check VERSION empty or unset case.
-  - INCLUDE="broken_no_access_rule" SERVER_EXPECTED_ERRORS="1"  # test errors are detected
+matrix:
+  include:
+    - python: 2.7
+      env:
+        VERSION="9.0" TESTS="1" LINT_CHECK="0"
+        INSTANCE_ALIVE="1" INCLUDE="broken_module" SERVER_EXPECTED_ERRORS="2"  # test errors are detected
+    - python: 2.7
+      env:
+        VERSION="9.0" TESTS="1" LINT_CHECK="0"
+        EXCLUDE="broken_module,broken_lint,broken_no_access_rule" OPTIONS="--log-level=debug" INSTALL_OPTIONS="--log-level=info" RUN_COMMAND_MQT_CREATE_FOLDER='mkdir /tmp/tests'
+    - python: 2.7
+      env:
+        VERSION="9.0" TESTS="1" LINT_CHECK="0"
+        INCLUDE="test_module,second_module" UNIT_TEST="1"
+    - python: 2.7
+      env:
+        VERSION="9.0" TESTS="1" LINT_CHECK="0"
+        TESTS="1" INCLUDE="test_module,second_module" ODOO_REPO="OCA/OCB" MQT_TEMPLATE_DB='mqt_odoo_template_core' MQT_TEST_DB='mqt_odoo_core'
+    - python: 2.7
+      env:
+        VERSION="9.0" TESTS="1" LINT_CHECK="0"
+        LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="19" TRAVIS_PULL_REQUEST="false" # Use main pylint config file
+    - python: 2.7
+      env:
+        VERSION="9.0" TESTS="1" LINT_CHECK="0"
+        VERSION=master EXCLUDE="broken_no_access_rule" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="32" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
+    - python: 2.7
+      env:
+        VERSION="9.0" TESTS="1" LINT_CHECK="0"
+        VERSION="7.0" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="30" TRAVIS_PULL_REQUEST="true"  # To check pylint_conf of PR's with old api
+    - python: 2.7
+      env:
+        VERSION="9.0" TESTS="1" LINT_CHECK="0"
+        VERSION="" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="19" TRAVIS_PULL_REQUEST="false"  # To check VERSION empty or unset case.
+    - python: 2.7
+      env:
+        VERSION="9.0" TESTS="1" LINT_CHECK="0"
+        INCLUDE="broken_no_access_rule" SERVER_EXPECTED_ERRORS="1"  # test errors are detected
+    - python: 3.4
+      env:
+        VERSION="9.0" TESTS="1" LINT_CHECK="0"
```

Comment:
Version 9.0?

Reply:
I'm not changing that part:
 - https://github.com/OCA/maintainer-quality-tools/blob/164f6801de47dfff42045a6c4216924621e97e1b/.travis.yml#L26



I just changed to use multi-python environment.

Reply:
Yeah, but 9.0 doesn't work with Python 3

Reply:
Oh, I got it your point.
Thanks for clarify.

We had a global configuration using `VERSION=9.0` but I didn't find a way to use a `global` configuration using multi-python platform. (overwritten from particular one)
Then I just copied and paste in one line the global configuration in order to see the same global effect and overwrite in the next line the particular one.

What do you think about?

Reply:
@pedrobaeza 
I just removed "global" environment variable and I'm using the required.